src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Ssl.cs

```diff
@@ -279,4 +290,86 @@ private SafeSslHandle() : base(IntPtr.Zero, true)
         {
         }
     }
+
+    internal sealed class SafeChannelBinding : SafeHandle
+    {
+        [StructLayout(LayoutKind.Sequential)]
+        private struct SEC_CHANNEL_BINDINGS
+        {
+            internal int dwInitiatorAddrType;
```

Comment:
We need a structure which is similar to SEC_CHANNEL_BINDINGS windows structure. So I kept the naming as it is. We can change it if required. let me know.